app/controllers/controller.py

```diff
@@ -0,0 +1,200 @@
+        for mail in mail_list:
+                if mail['mail_id'] == key and (mail['senderID'] == current_Uid['user_id'] or mail['recieverId'] == current_Uid['user_id']):
+                    mail_list.remove(mail)
+                    return jsonify({
+                        'status': 200,
+                        'data': [Static_strings.msg_deleted]
+                    })
+        return jsonify({
+            'status': 204,
+            'error': Static_strings.error_missing
+        })
+
+    def mail_send(self):
+        if not request.json:
+            return jsonify({
+                'status': 417,
+                'error': Static_strings.error_bad_data
+            })
+        mail_details = request.get_json()
+        if 'sen_status' not in mail_details:
+            return jsonify({
+                'status': 417,
+                'error': Static_strings.error_savemode
+            })
+        if mail_details['sen_status']=='sent' and 'recieverId' not in mail_details:
+            return jsonify({
+                'status': 417,
+                'error': Static_strings.error_missdestination
+            })
+        
+        subject = mail_details.get("subject")
+        parentMessageId = mail_details.get("parentMessageId")
+        sen_status = mail_details.get("sen_status")
+        senderID = mail_details.get("senderID")
+        recieverId = mail_details.get("recieverId")
+        msgdetails = mail_details.get("msgdetails")
+
+        new_mail = Mail(
+            subject=subject,
+            parentMessageId= parentMessageId,
+            sen_status= sen_status,
+            senderID= senderID,
+            recieverId= recieverId,
+            msgdetails= msgdetails
+        )
+
+        mail_list.append(
+            new_mail.mail_struct()
+        )
+        return jsonify({
+            'status': 201,
+            'data': [
+                new_mail.mail_struct()
+            ]
+        })
+
+    def accreate(self):
+        if not request.json:
+            return jsonify({
+                'status': 417,
+                'error': Static_strings.error_bad_data
+            })
+        user_details = request.get_json()
+        if 'email_add' not in user_details or 'first_name' not in user_details or 'last_name' not in user_details or 'password' not in user_details:
+            return jsonify({
+                'status': 403,
+                'error': Static_strings.error_bad_data
+            })
+        
+        email_add = user_details.get("email_add")
+        first_name = user_details.get("first_name")
+        last_name = user_details.get("last_name")
+        password = user_details.get("password")
+
+        new_user = User(
+            email_add=email_add,
+            first_name= first_name,
+            last_name= last_name,
+            password= password
+        )
+
+        user_list.append(
+            new_user.user_struct()
+        )
+        tok = create_access_token(new_user.user_struct()['user_id'])
```

Comment:
rename this to token. token is not so long to be cut short. 
you should ensure that your code is written in a way that it makes it easy for a person to understand what is taking place. though your variable naming and method naming

Reply:
Alright I'm on it

Reply:
This has been resolved